### PR TITLE
New branding properties

### PR DESCRIPTION
**Pull request by AlexanderMars**
- Added new properties to the property list in the resource and UPDATE pages
- Updated the first examples in LIST localizations, GET branding, and GET localization
- Changelog is not updated yet
- Added "hideAccountResetCredentials" according to the Pull Request 5352767: Exposing hideAccountResetCredentials property in PPE

### CODE REVIEW

**Comment by FaithOmbongi** [comment]
@AlexanderMars Once the PR is complete, passed validation, and is ready for content review, please remember to add the **ready for content review** label.

File: api-reference/beta/resources/organizationalbranding.md [hunk 1]
```diff
@@ -73,9 +88,24 @@ The following is a JSON representation of the resource.
...
   "signInPageText": "String",
   "squareLogo": "Stream",
   "squareLogoRelativeUrl": "String",
-  "usernameHintText": "String"
+  "usernameHintText": "String",
+  "loginPageTextVisibilitySettings": {
+    "hideAccountResetCredentials": "boolean",
+    "hideTermsOfUse": "boolean",
+    "hidePrivacyAndCookies": "boolean"
+  }
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/organizationalbranding.md:109]: This should resolve an API doctor error
```suggestion
  "loginPageTextVisibilitySettings": {
    "@odata.type": "microsoft.graph.loginPageTextVisibilitySettings"
  }
```

File: api-reference/beta/resources/organizationalbrandinglocalization.md [hunk 2]
```diff
@@ -66,9 +80,24 @@ The following is a JSON representation of the resource.
...
   "signInPageText": "String",
   "squareLogo": "Stream",
   "squareLogoRelativeUrl": "String",
-  "usernameHintText": "String"
+  "usernameHintText": "String",
+  "loginPageTextVisibilitySettings": {
+    "hideAccountResetCredentials": "boolean",
+    "hideTermsOfUse": "boolean",
+    "hidePrivacyAndCookies": "boolean"
+  }
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/organizationalbrandinglocalization.md:101]: For API doctor
```suggestion
  "loginPageTextVisibilitySettings": {
    "@odata.type": "microsoft.graph.loginPageTextVisibilitySettings"
  }
```

File: api-reference/beta/resources/organizationalbrandingproperties.md [hunk 3]
```diff
@@ -66,10 +80,25 @@ The following is a JSON representation of the resource.
...
   "signInPageText": "String",
   "squareLogo": "Stream",
   "squareLogoRelativeUrl": "String",
-  "usernameHintText": "String"
+  "usernameHintText": "String",
+  "loginPageTextVisibilitySettings": {
+    "hideAccountResetCredentials": "boolean",
+    "hideTermsOfUse": "boolean",
+    "hidePrivacyAndCookies": "boolean"
+  }
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/organizationalbrandingproperties.md:101]: For API doctor
```suggestion
  "loginPageTextVisibilitySettings": {
    "@odata.type": "microsoft.graph.loginPageTextVisibilitySettings"
  }
```

File: api-reference/beta/resources/organizationalbrandingproperties.md [hunk 4]
```diff
@@ -36,14 +36,28 @@ None.
...
 | signInPageText | String | Text that appears at the bottom of the sign-in box. You can use this to communicate additional information, such as the phone number to your help desk or a legal statement. This text must be Unicode and not exceed 1024 characters. |
 | squareLogo | Stream | A square version of your company logo that appears in Windows 10 out-of-box experiences (OOBE) and when Windows Autopilot is enabled for deployment. Allowed types are PNG or JPEG no larger than 240 x 240 pixels and no more than 10 KB in size. We recommend using a transparent image with no padding around the logo. |
 | squareLogoRelativeUrl | String | A relative url for the **squareLogo** property that is combined with a CDN base URL from the **cdnList** to provide the version served by a CDN. Read-only. |
 | usernameHintText | String | String that shows as the hint in the username textbox on the sign-in screen. This text must be a Unicode, without links or code, and can't exceed 64 characters. |
 
 ## Relationships
-None.
+| text visibility | [loginPageTextVisibilitySettings](../resources/loginpagetextvisibilitysettings.md) complex type | Represent the various texts that can be hidden. |
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/organizationalbrandingproperties.md:60]: Doesn't exist in the public metadata

File: api-reference/beta/resources/organizationalbrandingproperties.md [hunk 5]
```diff
@@ -36,14 +36,28 @@ None.
...
+| favicon | Stream | A custom browser icon (favicon) to replace a default “Microsoft logo” value utilizing AAD Company Branding blade. |
+| faviconRelativeUrl | String | A relative url for the favicon above that is combined with a CDN base url in cdnList to provide the version served by a CDN. Read-only. |
+| headerBackgroundColor | String | String containing RGB color that will enable admins customize the color of the header. |
 | id | String | An identifier that represents the locale specified in the ISO 639-1 standard, for example English is `en-US`. The **id** for the default /branding is always the String types `0` or `default`. Read-only. <br/><br/>**NOTE:** Multiple branding for a single locale are currently not supported. |
+| loginPageTextVisibilitySettings | [loginPageTextVisibilitySettings](../resources/loginPageTextVisibilitySettings.md) | This is a complex type that represents the various texts that can be hidden on the login page for a tenant. Inherited from [loginPageTextVisibilitySettings](../resources/loginpagetextvisibilitysettings.md). |
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/organizationalbrandingproperties.md:50]:
```suggestion
| loginPageTextVisibilitySettings | [loginPageTextVisibilitySettings](../resources/loginPageTextVisibilitySettings.md) | This is a complex type that represents the various texts that can be hidden on the login page for a tenant. |
```

File: api-reference/beta/resources/organizationalbrandinglocalization.md [hunk 6]
```diff
@@ -35,14 +35,28 @@ Inherits from [organizationalBrandingProperties](../resources/organizationalbran
...
+| headerBackgroundColor | String | String containing RGB color that will enable admins customize the color of the header. Inherited from [organizationalBrandingProperties](../resources/organizationalbrandingproperties.md). |
 | id | String | An identifier that represents the locale specified in the ISO 639-1 standard, for example English is `en-US`. The **id** for the default /branding is always the String types `0` or `default`. Read-only. <br/><br/>**NOTE:** Multiple branding for a single locale are currently not supported. Inherited from [organizationalBrandingProperties](../resources/organizationalbrandingproperties.md). |
+| loginPageTextVisibilitySettings | [loginPageTextVisibilitySettings](../resources/loginPageTextVisibilitySettings.md) | This is a complex type that represents the various texts that can be hidden on the login page for a tenant. Inherited from [loginPageTextVisibilitySettings](../resources/loginpagetextvisibilitysettings.md). |
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/organizationalbrandinglocalization.md:49]: Property is defined in organizationalBrandingProperties resource. So it's inherited from that resource.
```suggestion
| loginPageTextVisibilitySettings | [loginPageTextVisibilitySettings](../resources/loginPageTextVisibilitySettings.md) | This is a complex type that represents the various texts that can be hidden on the login page for a tenant. Inherited from [organizationalBrandingProperties](../resources/organizationalbrandingproperties.md). |
```

File: api-reference/beta/resources/organizationalbranding.md [hunk 7]
```diff
@@ -35,21 +35,36 @@ You can't change your original configuration's language. However, companies can
...
 | squareLogoRelativeUrl | String | A relative url for the **squareLogo** property that is combined with a CDN base URL from the **cdnList** to provide the version served by a CDN. Read-only. Inherited from [organizationalBrandingProperties](../resources/organizationalbrandingproperties.md). |
 | usernameHintText | String | String that shows as the hint in the username textbox on the sign-in screen. This text must be a Unicode, without links or code, and can't exceed 64 characters. Inherited from [organizationalBrandingProperties](../resources/organizationalbrandingproperties.md). |
 
 ## Relationships
 |Relationship|Type|Description|
 |:---|:---|:---|
 | localizations | [organizationalBrandingLocalization](../resources/organizationalbrandinglocalization.md) collection | Add different branding based on a locale. |
+| text visibility | [loginPageTextVisibilitySettings](../resources/loginpagetextvisibilitysettings.md) complex type | Represent the various texts that can be hidden. |
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/organizationalbranding.md:67]: Doesn't exist in the public metadata

File: api-reference/beta/resources/organizationalbranding.md [hunk 8]
```diff
@@ -35,21 +35,36 @@ You can't change your original configuration's language. However, companies can
...
+| headerBackgroundColor | String | String containing RGB color that will enable admins customize the color of the header. Inherited from [organizationalBrandingProperties](../resources/organizationalbrandingproperties.md). |
 | id | String | An identifier that represents the locale specified in the ISO 639-1 standard, for example English is `en-US`. The **id** for the default /branding is always the String types `0` or `default`. Read-only. <br/><br/>**NOTE:** Multiple branding for a single locale are currently not supported. Inherited from [organizationalBrandingProperties](../resources/organizationalbrandingproperties.md). |
+| loginPageTextVisibilitySettings | [loginPageTextVisibilitySettings](../resources/loginPageTextVisibilitySettings.md) | This is a complex type that represents the various texts that can be hidden on the login page for a tenant. Inherited from [loginPageTextVisibilitySettings](../resources/loginpagetextvisibilitysettings.md). |
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/organizationalbranding.md:54]: Same comment as before
```suggestion
| loginPageTextVisibilitySettings | [loginPageTextVisibilitySettings](../resources/loginPageTextVisibilitySettings.md) | This is a complex type that represents the various texts that can be hidden on the login page for a tenant. Inherited from [organizationalBrandingProperties](../resources/organizationalbrandingproperties.md). |
```

File: api-reference/beta/resources/loginpagetextvisibilitysettings.md [hunk 9]
```diff
@@ -0,0 +1,43 @@
...
+Namespace: microsoft.graph
+
+[!INCLUDE [beta-disclaimer](../../includes/beta-disclaimer.md)]
+
+This is a complex type that represents the various texts that can be hidden on the login page for a tenant.
+
+## Properties
+|Property|Type|Description|
+|:---|:---|:---|
+| loginPageTextVisibilitySettings | [loginPageTextVisibilitySettings](../resources/loginPageTextVisibilitySettings.md) | This is a complex type that represents the various texts that can be hidden on the login page for a tenant. |
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/loginpagetextvisibilitysettings.md:21]: Not a property. But we're missing some 2 other properties and hideAccountResetCredentials should be **hideResetItNow**

image

```suggestion
```

File: api-reference/beta/resources/loginpagetextvisibilitysettings.md [hunk 10]
```diff
@@ -0,0 +1,43 @@
...
+  "loginPageTextVisibilitySettings": {"@odata.type": "microsoft.graph.loginPageTextVisibilitySettings"}
+}
+-->
+``` json
+{
+  "@odata.type": "#microsoft.graph.loginPageTextVisibilitySettings",
+  "hideAccountResetCredentials": "Boolean",
+  "hidePrivacyAndCookies": "Boolean",
+  "hideTermsOfUse": "Boolean"
+}
```
Comment (FaithOmbongi) [inline api-reference/beta/resources/loginpagetextvisibilitysettings.md:42]: Same spirit as the Proeprties table
```suggestion
{
  "@odata.type": "#microsoft.graph.loginPageTextVisibilitySettings",
  "hideCannotAccessYourAccount": "Boolean",
  "hideForgotMyPassword": "Boolean",
  "hideResetItNow": "Boolean",
  "hidePrivacyAndCookies": "Boolean",
  "hideTermsOfUse": "Boolean"
}
```

**Review by FaithOmbongi — CHANGES_REQUESTED** [review]
@AlexanderMars left a few comments mostly related to the API doctor error.

File: api-reference/beta/resources/loginpagetextvisibilitysettings.md (hunk 9 above)
Comment (AlexanderMars) [reply api-reference/beta/resources/loginpagetextvisibilitysettings.md]: We have another PR to add "hideAccountResetCredentials" https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/5352767
This is a single replacement property for those I've removed - hideResetItNow, hideForgotMyPassword and hideCannotAccessYourAccount.

File: api-reference/beta/resources/loginpagetextvisibilitysettings.md (hunk 10 above)
Comment (AlexanderMars) [reply api-reference/beta/resources/loginpagetextvisibilitysettings.md:41]: We have another PR to add "hideAccountResetCredentials" https://msazure.visualstudio.com/One/_git/AD-AggregatorService-Workloads/pullrequest/5352767
This is a single replacement property for those I've removed - hideResetItNow, hideForgotMyPassword and hideCannotAccessYourAccount.

**Review by FaithOmbongi — APPROVED** [review]

**Pull request merged**